README.md

```diff
@@ -278,6 +279,47 @@ Configuration is done by user application: it should configure exporter and may
     });
     ```
 
+### Configuration with ASP.NET (Full .NET Framework) running in IIS
+
+1. Add a reference to the `OpenTelemetry.Collector.AspNet` package. Add any other collectors & exporters you will need.
+
+2. Add the Microsoft telemetry module in your `Web.config`:
+    ```xml
+    <system.webServer>
+        <modules>
+          <add name="TelemetryCorrelationHttpModule" type="Microsoft.AspNet.TelemetryCorrelation.TelemetryCorrelationHttpModule, Microsoft.AspNet.TelemetryCorrelation" preCondition="integratedMode,managedHandler"/>

```

Comment:
is this module responsible for parsing incoming trace headers, populate activity etc?

Reply:
Ya. I took inspiration from AppInsights with that.